### ISSUE TITLE
Invalid object name 'dbo.__MigrationHistory'

### ISSUE THREAD

**Issue by b4naki** [thepirat000/Audit.NET#237]
When debugging in Visual Studio we are getting the following Exception pretty often: 
```
"System.Data.SqlClient.SqlException" in System.Data.dll
Invalid object name 'dbo.__MigrationHistory'.
```
Entity Framework is right here we don't use 'dbo.__MigrationHistory' and we dont use Code First Migrations. We are only using EF for Audit.

After a little investigation it seems that one can turn off the check for  'dbo.__MigrationHistory' by setting 
`Database.SetInitializer<YourContext>(null);`
according to this StackOverflow [Link:] 
(https://stackoverflow.com/questions/20877039/invalid-object-name-dbo-migrationhistory-using-database-create-ef6-02-when)

is there a way to tell the 
`Audit.Core.Configuration.DataProvider` 
to use 
`Database.SetInitializer<YourContext>(null);` ? 
Or another way to stop the cause of the exception?

Heres the Code to create the audit scope 
```
` private void SetAuditProvider(DirectDbDataAccess db)
        {
            Audit.Core.Configuration.DataProvider = new SqlDataProvider()
            {
                ConnectionString = db.ConnectionInfo.ConnectionString,
                Schema = "dbo",
                TableName = "Audit",
                IdColumnName = "EventId",
                JsonColumnName = "Data",
                LastUpdatedDateColumnName = "LastUpdatedDate",
                CustomColumns = new List<CustomColumn>()
                {
                    new CustomColumn("EventType", ev => ev.EventType)
                }
            };

            Audit.Core.Configuration.AddCustomAction(ActionType.OnScopeCreated, scope =>
            {
                if (!m_audit.isActive()) scope.Discard();
            });
        }`
```

heres the code to create the table: 

```
CREATE TABLE [Audit]
                (
	                [EventId] BIGINT IDENTITY(1,1) NOT NULL,
	                [InsertedDate] DATETIME NOT NULL DEFAULT(GETUTCDATE()),
	                [LastUpdatedDate] DATETIME NULL,
	                [Data] NVARCHAR(MAX) NOT NULL,
                        [EventType] NVARCHAR(100) NOT NULL,
	                CONSTRAINT PK_Event PRIMARY KEY (EventId)
                )
```

**Comment by thepirat000** [comment]
The only way I can think of providing such option, will be to have a boolean configuration value to specify if the initializer should be set to `NULL` on the constructor of the internal [`AuditContext`](https://github.com/thepirat000/Audit.NET/blob/master/src/Audit.NET.SqlServer/AuditContext.cs). 

I can't find a more generic way to provide an initialization function without changing `AuditContext` accessibility

**Comment by b4naki** [comment]
Both options would work for us.

**Comment by thepirat000** [comment]
ok I will release a version with the first option soon

**Comment by thepirat000** [comment]
This is included starting on version 14.6.3

**Comment by b4naki** [comment]
I updated this moring. works great. thank you for help and your efforts you put into Audit.net :)
